### PR TITLE
Update cudd.pyx

### PR DESCRIPTION
**Pull request by SaiCharanMarrivada**

### CODE REVIEW

**Comment by SaiCharanMarrivada** [comment]
I found that bdd.var('x') ^ bdd.var('y') is much faster compared to bdd.add_expr('x ^ y'). For instance, later took 14.6us while former took only 300ns.

**Comment by slivingston** [comment]
@SaiCharanMarrivada Can you tell me why you continue to push so many "Update (filename)" commits? Or, tell me when this is ready for review.

**Comment by SaiCharanMarrivada** [comment]
@slivingston Sorry, No more commits and it is ready for review. Thanks for such a nice package in python, it is very helpful for my project.

File: dd/cudd.pyx [hunk 1]
```diff
@@ -2243,6 +2266,13 @@ cdef class Function(object):
         r = Cudd_bddOr(self.manager, self.node, other.node)
         return wrap(self.bdd, r)
 
+    def __xor__(Function self, Function other):
```
[the remark this answers is not in the document]
Comment (SaiCharanMarrivada) [reply dd/cudd.pyx:2269]: Oh, I am not aware of that. I feel using xor operator is more intuitive.

**Comment by slivingston** [comment]
@SaiCharanMarrivada This PR needs work, in particular:

* There are a bunch of little commits without clear commit messages.
* [A key question](https://github.com/tulip-control/dd/pull/90#discussion_r1421565477) is not answered: "In which cases have reordering methods other than sifting been measured faster ?"
* There should be more documentation or examples that demonstrate your proposed change.
* A concise description for the pull request would help. (Currently [it is empty](https://github.com/tulip-control/dd/pull/90#issue-1824007058).)

I am going to close the PR assuming it is abandoned, but feel free to re-open or send a new PR, considering my feedback.

**Pull request closed without merging**